src/Hint/ListRec.hs

```diff
@@ -51,6 +51,7 @@ listRecHint _ _ = concatMap f . universe
             return $ idea severity ("Use " ++ use) o y [Replace Decl (toSS o) [] (prettyPrint y)]
 
 
+recursiveStr :: String
```

Comment:
Not sure this one is worth it. Don't care overly though.

Reply:
I'm trying to get it clean of warnings with `-Wmissing-sigs` enabled.

Reply:
Will you ever do that with the huge number of `isFoo` methods in the HSE.Utils?

Reply:
Heh :) Sounds like I've created a switch to beat myself with. I guess I'll desist then.